src/src/com/microsoft/aad/adal/ADALError.java

```diff
@@ -466,6 +466,11 @@
      * WebView returned empty redirect url.
      */
     WEBVIEW_RETURNED_EMPTY_REDIRECT_URL("Webview returned empty redirect url"),
+    
+    /**
+     * WebView  redirect url is not SSL protected.
+     */
+    WEBVIEW_REDIRECTURL_NOT_SSL_PROTECTED("Webview redirect url is not SSL protected"),
```

Comment:
I suspect this would be too easily confused with the "redirect uri" parameter, something the developer has control over. In this case the issue is with the server configuration. "The webview was redirected to an unsafe URL" would be a little more clear on that point.